### ISSUE TITLE
[9.0 Staging-1477] North/South Axis of Map Unclaimable Claims

### ISSUE THREAD

**Issue by Sarah-Graesyn** [StrangeLoopGames/EcoIssues#15268]
After setting up some zones and an Infrastructure Director on Royal Otter I noticed last night that there is a 2 claim wide band of claims that you cannot claim using the new map editor for existing claims and then if you manually claim them each claim you manually claim forces the creation of a new plot.

The center coordinates for a handful of those claims are:
467x, 851z 
467x, 846z 
462x, 847z
462x, 852z

This change disrupted the ability to protect my holdings under one claim but its more important that it seems like a bug from  something. In the screenshot below each claim is highlighted in a different color for contrast.

![20200323043940_1](https://user-images.githubusercontent.com/1329717/77317181-b5612600-6cd8-11ea-8dd1-4b450a0dca1f.jpg)

**Comment by SlayksWood** [comment]
@Daelmaron 
Can you describe it more?

**Comment by Daelmaron** [comment]
@SlayksWood not sure that I have a whole lot more detail on it, also not entirely sure what actually caused it.

I noticed it when I was trying to expand my claim and there was a band of claims that I suddenly couldn't interact with on the map method of claiming/unclaiming and then when I went to go investigate it with the claim stake the screenshot of the 5 different claims is what resulted. 

For whatever reason I cannot add those 4 (the multicolored) claims to the larger (blue) claim and make them one, claiming them forces the creation of a brand new claim, which you also cannot merge with one another, each claim forces its own new one.
